### ISSUE TITLE
Webdesign : a lone serif font paragraph

### ISSUE THREAD

**Issue by Facyla** [linagora/tmail-flutter#2620]
On FF web / Ubuntu
When accessing Settings > Identities : the text on the right is in a serif font which does not match the global UI 
![image](https://github.com/linagora/tmail-flutter/assets/286492/2595483a-a3e7-495a-9eb4-368c679d6ca4)

**Comment by Facyla** [comment]
nope, it happens consistently when i created fresh new identities to reproduce the issue.

Steps to reproduce: head to Settings > create new identity > Select the newly created identity.

**Comment by hoangdat** [comment]
Did you setup signature before in other platform?

**Comment by Facyla** [comment]
I did not set explicitly any signature. After noticing this i created 2 other identities and the exact same behaviour happened.

**Comment by chibenwa** [comment]
Is this still happening when creating a new identity?
If so please reopen
